Umbraco-Heartcore/API-Documentation/index.md

```diff
@@ -4,25 +4,25 @@ versionFrom: 8.0.0
 
 # API Documentation
 
-This page contains documentation for the Umbraco Headless API endpoints divided into two main areas: Content Delivery and Content Mangement.
+This page contains documentation for the REST API endpoints for Umbraco Heartcore. It is divided into two main areas: Content Delivery and Content Mangement.
 
 The [Content Delivery API](Content-Delivery) is the read-only Content and Media that you would normally retrieve to show the published content in your apps, websites or other platforms. The API is available on `https://cdn.umbraco.io`.
 
 [The Content Management API](Content-Management) can be used to Create, Read, Update and Delete Content, Media, Languages, Relations, Members and the associated types using Umbraco Backoffice user credentials or API Keys. The API is available on `https://api.umbraco.io`.
 
 ## REST API Standard
 
-The Umbraco Headless APIs are based on the [HAL Standard](https://weierophinney.github.io/hal/hal/).
+The REST APIs are based on the [HAL Standard](https://weierophinney.github.io/hal/hal/).
 
 ### System level properties
 
-The properties in the Umbraco Headless API, which starts with an underscore, are **system level properties**. That means that they are standard Umbraco properties, which cannot be changed via the API. This includes properties like `_id`, `_url`, `_createDate`, `_updateDate`, `_creatorName`, `_writerName`, `_level` and `_hasChildren`. These are all defined by Umbraco when Content is created or updated.
+The properties in the REST API, which starts with an underscore, are **system level properties**. That means that they are standard Umbraco properties, which cannot be changed via the API. This includes properties like `_id`, `_url`, `_createDate`, `_updateDate`, `_creatorName`, `_writerName`, `_level` and `_hasChildren`. These are all defined by Umbraco when Content is created or updated.
 
-The properties `_links` and `_embedded` are both part of the HAL specification and are implemented in the Umbraco Headless API accordingly.
+The properties `_links` and `_embedded` are both part of the HAL specification and are implemented in the REST API accordingly.
 
 ### API Browser
 
-In the Settings section in the Umbraco Backoffice, there is a Headless tree. From there you can use the API Browser to interact with both the Content Delivery and Content Management APIs. 
+In the Settings section in the Umbraco Backoffice, there is a Heartcore specific tree. From there you can use the API Browser to interact with both the Content Delivery and Content Management APIs. 
```

Comment:
Not sure about renaming the tree node

Reply:
Let's leave it as "Headless" for now then.